docs/posts/flink-quickstart/index.qmd

```diff
@@ -0,0 +1,259 @@
+---
+title: "Ibis goes real-time! Introducing the new Flink backend for Ibis"
+author: "Deepyaman Datta"
+date: "2024-01-10"
+categories:
+    - blog
+    - flink
+    - stream processing
+---
+
+## Introduction
+
+Ibis 8.0.0 marks the official release of the Apache Flink backend for Ibis. Flink is one of the most established stream-processing frameworks out there and a central part of the real-time data infrastructure at companies like DoorDash, LinkedIn, Netflix, and Uber. The Flink backend is also the first streaming backend Ibis supports. Follow along as we define and execute a simple streaming job using Ibis!
+
+## Installation prerequisites
+
+* **Docker Compose:** This tutorial uses Docker Compose to manage an Apache Kafka environment (including sample data generation) and a Flink cluster (for [remote execution](#remote-execution)). You can [download and install Docker Compose from the official website](https://docs.docker.com/compose/install/).
+* **JDK 11:** Flink requires Java 11. If you don't already have JDK 11 installed, you can [get the appropriate Eclipse Temurin release](https://adoptium.net/temurin/releases/?package=jdk&version=11).
+* **Python:** To follow along, you need Python 3.9 or 3.10.
+
+## Installing the Flink backend for Ibis
+
+We use a Python client to explore data in Kafka topics. You can install it, alongside the Flink backend for Ibis, with `pip`, `conda`, `mamba`, or `pixi`:
+
+::: {.panel-tabset}
+
+## Using `pip`
+
+```bash
+pip install ibis-framework apache-flink kafka-python
+```
+
+## Using `conda`
+
+::: {.callout-important}
+## PyFlink is not available on conda-forge; please use `pip` to install the PyFlink backend instead.
+:::
+
+## Using `mamba`
+
+::: {.callout-important}
+## PyFlink is not available on conda-forge; please use `pip` to install the PyFlink backend instead.
+:::
+
+## Using `pixi`
+
+::: {.callout-important}
+## PyFlink is not available on conda-forge; please use `pip` to install the PyFlink backend instead.
+:::
+
+:::
+
+## Spinning up the services using Docker Compose
+
+The [claypotai/ibis-flink-example GitHub repository](https://github.com/claypotai/ibis-flink-example) includes the relevant Docker Compose configuration for this tutorial. Clone the repository, and run `docker compose up` from the cloned directory to create Kafka topics, generate sample data, and launch a Flink cluster.
+
+```bash
+git clone https://github.com/claypotai/ibis-flink-example.git
+cd ibis-flink-example
+docker compose up
+```
+
+::: {.callout-tip}
+If you don't intend to try [remote execution](#remote-execution), you can start only the Kafka-related services with `docker compose up kafka init-kafka data-generator`.
+:::
+
+After a few seconds, you should see messages indicating your Kafka environment is ready:
+
+```bash
+ibis-flink-example-init-kafka-1      | Successfully created the following topics:
+ibis-flink-example-init-kafka-1      | payment_msg
```

Comment:
It's jarring to see a reference to "payment" out of context like this. If I were running this on hosted infrastructure, I might have a small panic attack upon seeing this, worried that I was incurring costs from my cloud provider. Of course, it's just a reference to the sample data. Can we include a sentence _above_ here explaining that this example uses payments data?

Reply:
[Added a line!](https://github.com/ibis-project/ibis/pull/7912/commits/691e3c72d50b5fc57043708e0194bf0cef105ab3)

Reply:
Yes I think this helps. Thanks!